instance/models/mixins/openedx_config.py

```diff
@@ -542,7 +557,57 @@ def _get_configuration_variables(self):
                 "EDXAPP_LMS_WRITABLE_GRADEBOOK_URL": "https://{}/gradebook".format(self.instance.mfe_domain),
                 "EDXAPP_PROFILE_MICROFRONTEND_URL": "https://{}/profile/u/".format(self.instance.mfe_domain),
             })
-
+        if self._is_openedx_release_in(['maple']):
+            template.update({
+                'EDXAPP_VIDEO_IMAGE_SETTINGS': {
+                    'VIDEO_IMAGE_MAX_BYTES': 2097152,
+                    'VIDEO_IMAGE_MIN_BYTES': 2048,
+                    'STORAGE_KWARGS': {
+                        'location': 'video-images/',
+                        'bucket': '{{ EDXAPP_AWS_STORAGE_BUCKET_NAME }}'
+                    }
+                },
+                'EDXAPP_VIDEO_TRANSCRIPTS_SETTINGS': {
+                    'VIDEO_TRANSCRIPTS_MAX_BYTES': 3145728,
+                    'STORAGE_KWARGS': {
+                        'location': 'video-transcripts/',
+                        'bucket': '{{ EDXAPP_AWS_STORAGE_BUCKET_NAME }}'
+                    }
+                },
+                'EDXAPP_CELERY_BROKER_TRANSPORT': 'amqp',
+                'MFE_BASE': self.instance.mfe_domain,
+                'MFE_SITE_NAME': self.instance.name,
+                'MFE_BASE_SCHEMA': 'https',
+                'MFE_DEPLOY_NGINX_PORT': 80,
+                'MFE_DEPLOY_COMMON_HOSTNAME': self.instance.mfe_domain,
+                'MFE_DEPLOY_VERSION': self.openedx_release,
+                'MFES': [
+                    {
+                        'name': 'gradebook',
+                        'repo': 'frontend-app-gradebook',
+                        'public_path': '/gradebook/'
+                    }
+                ],
+                'EDXAPP_CSRF_COOKIE_SECURE': True,
+                'EDXAPP_SESSION_COOKIE_SECURE': True,
+                'EDXAPP_ENABLE_CORS_HEADERS': True,
+                'EDXAPP_ENABLE_CROSS_DOMAIN_CSRF_COOKIE': True,
+                'EDXAPP_CROSS_DOMAIN_CSRF_COOKIE_DOMAIN': '.{}'.format(self.instance.domain),
+                'EDXAPP_CROSS_DOMAIN_CSRF_COOKIE_NAME': 'cross-domain-cookie-mfe',
+                'EDXAPP_CORS_ORIGIN_WHITELIST': [
+                    self.instance.url.rstrip('/'),
+                    self.instance.lms_preview_url.rstrip('/'),
+                    self.instance.studio_url.rstrip('/'),
+                    self.instance.mfe_url.rstrip('/'),
+                ],
+                'EDXAPP_LMS_WRITABLE_GRADEBOOK_URL': '{}/gradebook'.format(self.instance.mfe_url),
```

Comment:
@viadanna, unless I'm missing something, this will resolve to `https://subdomain.domain//gradebook` (double slash).